ignite-3/src/jepsen/ignite3.clj

```diff
@@ -27,22 +31,35 @@
       (for [n other-nodes] (str "\"" n ":3344\""))
       ["\"localhost:3344\""])))
 
+(defn node-name
+  "Generates a default name for the given node."
+  [all-nodes current-node]
+  (str "node-" (inc (.indexOf all-nodes current-node))))
+
 (defn configure-server!
   "Creates a server config file and uploads it to the given node."
-  [test all-nodes current-node]
-  (let [content (list-nodes all-nodes current-node)
-        replace-command (str "s/\"localhost:3344\"/" (clojure.string/join ", " content) "/")]
-    (c/exec :sed :-i replace-command (str (db-dir test) "/etc/ignite-config.conf"))))
+  [test node]
+  (let [all-nodes (:nodes test)]
+    (c/exec :sed :-i (str "s/defaultNode/" (node-name all-nodes node) "/")
+                     (db-dir test "etc" "vars.env"))
+    (c/exec :sed :-i (str "s/\"localhost:3344\"/" (clojure.string/join ", " (list-nodes all-nodes node)) "/")
+                     (db-dir test "etc" "ignite-config.conf"))))
 
 (defn start!
   "Starts server for the given node."
-  [node test]
+  [test node]
   (info node "Starting server node")
   (c/cd (db-dir test) (c/exec "bin/ignite3db" "start"))
   (Thread/sleep 3000)
-  (c/cd (cli-dir test)
-        (c/exec "bin/ignite3" "cluster" "init" "--cluster-name=ignite-cluster" "--meta-storage-node=defaultNode"))
-  (Thread/sleep 3000))
+  (when (= 0 (.indexOf (:nodes test) node))
+    (info node "Init cluster")
+    (c/cd (cli-dir test)
+          (c/exec "bin/ignite3"
+                  "cluster"
+                  "init"
+                  "--cluster-name=ignite-cluster"
+                  (str "--meta-storage-node=" (node-name (:nodes test) node))))
```

Comment:
Currently, a 1 node is used as meta storage node and as cluster management group member. Probably, we should add all nodes into CMG. WDYT, folks?

Reply:
> we should add all nodes into CMG.
I believe this would make sense when we will have scenarios with enabled nemesis. But right now let's leave the current implementation.